src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -435,7 +428,7 @@ class InvoiceForm extends Component {
                     </Row>
                   </Accordion> */}
                   <Accordion
-                    id={SECTIONS.documents}
+                    id={SECTIONS.DOCUMENTS}
```

Comment:
`SECTIONS.documents` was looking better, `SECTIONS` is a constant, `documents` is just a key

Reply:
I'll switch keys to lowerCase